numpy/lib/histograms.py

```diff
@@ -345,6 +345,99 @@ def _search_sorted_inclusive(a, v):
         a.searchsorted(v[-1:], 'right')
     ))
 
+def histogram_bin_edges(a, bins=10, range=None, weights=None):
```

Comment:
nit: extra newline needed above the `def` for PEP8

Reply:
Done.